src/openrpc/discovery.json

```diff
@@ -1253,6 +1253,43 @@
 				}
 			]
 		},
+		{
+			"name": "getNavigationIntent",
+			"tags": [
+				{
+					"name": "property:immutable"
+				},
+				{
+					"name": "capabilities",
+					"x-uses": [
+						"xrn:firebolt:capability:discovery:navigate-to"
+					]
+				}
+			],
+			"summary": "get the latest navigation intent",
+			"params": [],
+			"result": {
+				"name": "intent",
+				"schema": {
+					"$ref": "https://meta.rdkcentral.com/firebolt/schemas/intents#/definitions/NavigationIntent"
+				}
+			},
+			"examples": [
+				{
+					"name": "Get the latest navigation intent",
+					"params": [],
+					"result": {
+						"name": "intent",
+							"value": {
```

Comment:
Mind fixing the spacing here?
If it helps, I use a VSCode extension to format/pretty-fy my documents.